src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -350,13 +369,32 @@ def lvm_pv_type_label(device)
         format(_("PV of %s"), vg.basename)
       end
 
-      # Type label when the device belongs to a multidevice Btrfs filesystem
+      # Type label when the device holds a journal
       #
       # @param filesystem [Y2Storage::Filesystems::Base]
       # @return [String]
-      def btrfs_multidevice_type_label(filesystem)
-        # TRANSLATORS: %s is a device base name. E.g., sda1+
-        format(_("Part of Btrfs %s"), filesystem.blk_device_basename)
+      def journal_type_label(filesystem)
+        # TRANSLATORS: %{fs_name} is the filesystem name. E.g., Btrfs, Ext4, etc.
+        #              %{blk_device_name} is a device base name. E.g., sda1+
+        format(
+          _("Journal of %{fs_name} %{blk_device_name}"),
+          fs_name:         filesystem.type,
+          blk_device_name: filesystem.blk_device_basename
+        )
```

Comment:
I would change this:

```
data_device = fs.devices.select { |d| !d.journal? }

format(_("%{fs_type} Journal (%{data_device_name})"), fs_type: fs.type.to_human_string,  data_device_name: data_device.basename)
```

With this we fix several things:

* We are using the translated label for the fs type
* We ensure to mention the data device (note that `#blk_device_basename` returns the first in alphabetical order).

And the result would something like: "Ext4 Journal (sda1)"